lib/inquisition/i18n_tasks/runner.rb

```diff
@@ -0,0 +1,32 @@
+require 'i18n/tasks/cli'
+
+module Inquisition
+  module I18nTasks
+    class Runner < ::Inquisition::Runner
+      def call
+        base_task = ::I18n::Tasks::BaseTask.new
+        errors = { missing_translate: base_task.missing_keys, unused: base_task.unused_keys }
+        parse_data_errors(errors)
+        @issues
+      end
+
+      private
+
+      def parse_data_errors(errors)
+        errors.each do |error_type, error|
+          error.keys.each { |key, node| @issues << create_issue(error_type, key, node) }
+        end
+      end
+
+      def create_issue(error_type, key, node)
+        Inquisition::Issue.new(
+          severity: :low,
+          path: node.data[:path],
+          line: nil,
```

Comment:
Based on the screen from the gem readme, this gem able to retrieve information about line for missing translations